### DISCUSSION TITLE
Add loading indicator when lists are refreshed

### DISCUSSION THREAD

**Discussion by tribut** [jonaswinkler/paperless-ng#1260] [Feature Requests]
When documents are filtered, there is no indication that something is happening until the results are displayed. I would love some sort of loading indicator (spinner?).

Especially when the result is empty(!) it is unclear when the search is finished. On the other hand with some longer-running queries I am often unsure whether I need to hit return again or if the search is already running.

This to some degree also applies to the lists on the dashboard.

**Comment by shamoon** [comment]
I like this idea, how about this (see video)? =)

https://user-images.githubusercontent.com/4887959/132621027-b31da84e-3603-48db-8e2b-2f98fd94edb6.mov
